### PR TITLE
support tf.math.round

### PR DESCRIPTION
**Pull request by nbcsm**
- support tf.math.round approximately
- fix loop_rewriter unused output shape

### CODE REVIEW

File: tf2onnx/tfonnx.py [hunk 1]
```diff
@@ -1638,6 +1638,24 @@ def where_op(ctx, node, name, args):
     return [node, transpose_node]
 
 
+def round_op7(ctx, node, name, args):
+    dtype = ctx.get_dtype(node.input[0])
```
Comment (pengwa) [inline tf2onnx/tfonnx.py]: add TF signature like we did for better understanding. 

"    # T output = Softmax(T logits). The axis softmax would be performed on is always on -1.
     # T output = Softmax(T input, @int axis). Default axis is 1."

File: tf2onnx/tfonnx.py [hunk 2]
```diff
@@ -1638,6 +1638,24 @@ def where_op(ctx, node, name, args):
...
+    if dtype in [onnx_pb.TensorProto.INT32, onnx_pb.TensorProto.INT64]:
+        node.type = "Identity"
+        return
+
+    # Workaround: for floating point, TF round mode is half to even, while round half up is used here.
+    # Will switch to onnx Round when it is ready.
+    # The difference now is for numbers like 2.5, 4.5 etc.
+    # In practise, it is supposed to have minor impact.
```
Comment (pengwa) [inline tf2onnx/tfonnx.py]: if there are model having *.5, we will get result mismatch. shall we avoid this?

Comment (pengwa) [reply tf2onnx/tfonnx.py]: it will not be a good experience to plan for a fix when we have a case with tons of efforts identifying it's this node causing mismatch.

Comment (nbcsm) [reply tf2onnx/tfonnx.py]: make sense, will close this pr.
